### PR TITLE
Install any country package

### LINKED ISSUE #92: Add deployment script for demo.openfisca.org/legislation

### PR DESCRIPTION
**Pull request by cbenz**
Follows #231 

This PR allows installing the Legislation Explorer working with any [country package](https://openfisca.org/en/countries/).

Fixes https://github.com/openfisca/openfisca-ops/issues/92.

### CODE REVIEW

File: ops/ansible/roles/legislation_explorer/defaults/main.yml [hunk 1]
```diff
@@ -26,7 +27,7 @@ matomo_url: null
 matomo_site_id: null
 
 # Fully managed by Ansible, you should not need to edit this unless you have a naming collision
-source_dir: "/home/{{ unix_user_name }}/legislation-explorer"
-systemd_service_file: /etc/systemd/system/legislation-explorer.service
+source_dir: "/home/{{ unix_user_name }}/legislation-explorer-{{ country_code }}"
+systemd_service_file: /etc/systemd/system/legislation-explorer-{{ country_code }}.service
```
Comment (MattiSG) [inline ops/ansible/roles/legislation_explorer/defaults/main.yml]: Then how is this cleaned up if I change the country package? 🙂

Comment (cbenz) [reply ops/ansible/roles/legislation_explorer/defaults/main.yml]: Cf https://github.com/openfisca/openfisca-ops/pull/104#issuecomment-953803482

**Review by MattiSG — APPROVED** [review]

**Comment by MattiSG** [comment]
Tested locally by changing the `api_url`

**Comment by cbenz** [comment]
@MattiSG Do you have any idea why the "Sauce Connect" steps fail?

**Comment by cbenz** [comment]
Restarting the CircleCI pipeline resulted in a success.

**Pull request merged**